nexus/social_media/services.py

```diff
@@ -0,0 +1,30 @@
+# Third Party Stuff
+import facebook
+from nexus.celery import app
+
+# Nexus Stuff
+from nexus.social_media.models import Post
+from nexus.base import response
+from settings.common import FB_USER_ACCESS_TOKEN, FB_PAGE_ID
```

Comment:
`from djang.conf import settings` and then do `settings.WHATEVER_VAR` in your code.